lib/core/components/empty-content/empty-content.component.scss

```diff
@@ -0,0 +1,32 @@
+@mixin adf-empty-content-theme($theme) {
+
+    $foreground: map-get($theme, foreground);
+
+    .adf-empty-content {
+        color: mat-color($foreground, text, 0.54);
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        &__icon {
+            font-size: 52px;
```

Comment:
can we use the typography variables instead of the hardcoded one?
i.e.
        typography.scss
        $display-1:     mat-typography-level(34px, 40px, 400),

       component.scss
          &-myclass {
            font-size: mat-font-size($config, display-1);